### PR TITLE
WG-wide merge guide

### PR DESCRIPTION
**Pull request by alvestrand**
Addresses the charter side of #63

### CODE REVIEW

File: merge-guide.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Procedure for adding new features to a REC-level WEBRTC specification
+
+The WebRTC-PC spec is now a W3C Rec, and the Mediacapture-Main spec
+is aiming for tht status.
```
Comment (jan-ivar) [inline merge-guide.md]:
```suggestion
is aiming for that status.
```

File: merge-guide.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+## Documenting implementations
+
+The chief mechanism to verify implementations is the Web Platform Tests suite.
+
+In order to be considered for integration, a new extension must have:
+
+* Reasonable test coverage
+* Two passing implementation
+
+## Merging to the REC branch
```
Comment (youennf) [inline merge-guide.md]: Merging to the Editor's draft?

Comment (alvestrand) [reply merge-guide.md]: Yes.

File: merge-guide.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+## Adding new features
+
+In order to have a new feature considered for inclusion, write it up as an extension spec
+or as a pull request against SPECNAME-extensions.
+
+For features that are "large" in some sense (large spec, need their own explainer, may have
+different people interested), extension specs are preferred. For extensions that are
+"small" (add a few methods, extend an argument list or enum, the function description
+is explanation enough), PRs against SPECNAME-extensions is preferred.
```
Comment (youennf) [inline merge-guide.md]: pull requests

**Pull request merged**